src/scripts/macros/hotbar.ts

```diff
@@ -46,13 +46,13 @@ export function rollItemMacro(itemId: string): ReturnType<ItemPF2e["toChat"]> |
     return item.toChat();
 }
 
-export async function createActionMacro(actionIndex: number, actorId: string, slot: number): Promise<void> {
+export async function createActionMacro(actionIndex: number, _actorId: string, slot: number): Promise<void> {
```

Comment:
We can change the parameters of this freely since it's only used to create new macros